src/main/java/io/confluent/connect/elasticsearch/ElasticsearchSinkConnectorConfig.java

```diff
@@ -737,6 +797,10 @@ public boolean isBasicProxyConfigured() {
     return !getString(PROXY_HOST_CONFIG).isEmpty();
   }
 
+  public boolean isDataStream() {
+    return dataStreamType() != DataStreamType.NONE && !dataStreamDataset().equals("");
```

Comment:
```suggestion
    return dataStreamType() != DataStreamType.NONE && !dataStreamDataset().isEmpty();
```

Reply:
Thanks, made a similar change in Validator.java too